### DISCUSSION TITLE
Is it possible to print the node/npm versions used by renovate within the PR?

### DISCUSSION THREAD

**Discussion by adguy360** [renovatebot/renovate#18889] [Request Help] [answered]
Is it possible to print the node/npm binary versions used by renovate when generating the PR? preferably printed within the PR comment.
This could serve as validation that the correct node/npm versions were picked up when updating dependencies.

I was also wondering if renovate should pick up the node version specified in my `.nvmrc` file?
With my self hosted (all default configs) this doesn't seem to work. Using either `engines` or `constraints` is the only per repo fix I could find.

Thanks

**Comment by rarkins** [comment, marked as answer]
I think this is overkill for every PR so don't support this as a feature as it increases complexity with only minimal benefit to the majority of users. The node and npm version used are available in logs. Do you have access to them?

**Reply by adguy360** [reply]
Thanks @rarkins, I don't have access to these logs, but I guess we could set something up in our company.
Regarding the `.nvmrc` usages - should it get auto detected and use the binary version from there?

**Reply by rarkins** [reply]
Yes, although it's been reported recently that the wrong node can be used if there's multiple different node versions (Eg via npmrc) in the same branch.
